pom.xml

```diff
@@ -565,26 +566,23 @@
       </modules>
     </profile>
 
+    <!-- Requires hadoop.version to be 2.4.0 or later -->
```

Comment:
How about adding an enforcer (in a separate PR) to make sure hadoop version is > 2.4.0.